wsdl/ver10/device/wsdl/devicemgmt.wsdl

```diff
@@ -2159,6 +2164,32 @@ IN NO EVENT WILL THE CORPORATION OR ITS MEMBERS OR THEIR AFFILIATES BE LIABLE FO
 							<xs:documentation>User credential for the storage server</xs:documentation>
 						</xs:annotation>
 					</xs:element>
+					<xs:element name="ConfigurationRenewal" minOccurs="0">
+						<xs:complexType>
+							<xs:sequence>
+								<xs:element name="RenewalEndpoint" type="xs:anyURI">
+									<xs:annotation>
+										<xs:documentation>Remote URL to be queried by the device to renew the storage configuration.</xs:documentation>
+									</xs:annotation>
+								</xs:element>
+								<xs:element name="AuthorizationServer" type="tt:ReferenceToken">
+									<xs:annotation>
+										<xs:documentation>JWTConfiguration token referring to an Authorization server that provides JWT token to authorize with the the renewal endpoint.</xs:documentation>
+									</xs:annotation>
+								</xs:element>
+								<xs:element name="CertPathValidationPolicyID" type="xs:string" minOccurs="0">
+									<xs:annotation>
+										<xs:documentation>CertPathValidationPolicyID used to validate the renewal endpoint server certificate. If CertPathValidationPolicyID is not configured, the certificate shall not be validated.</xs:documentation>
+									</xs:annotation>
+								</xs:element>
+								<xs:element name="Error" type="xs:string" minOccurs="0">
```

Comment:
When is this going to be used?

Reply:
@jmelancongen I just approved #424 and I agree with @ubkr please re-review that PR and update this accordingly. Many kind thanks.

Reply:
Oh, accidentally resolved the comment early.
This follows the recent PR from Hans #419 so we have at least available information in case the renewal flow breaks unexpectedly. The client will be able to retrieve that error and help narrow down the issue.